app/views/candidate_interface/new_references/review/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<% content_for :title, t('page_titles.new_references') %>
+<% content_for :before_content, govuk_back_link_to(candidate_interface_application_form_path) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">
```

Comment:
We should add the error summary here, above the h1, instead of above the form below.